src/imio/esign/tests/test_utils.py

```
@@ -2,9 +2,11 @@
 """utils tests for this package."""
 from datetime import date
 from datetime import timedelta
+from imio.esign.config import set_registry_max_session_size
 from imio.esign.testing import IMIO_ESIGN_INTEGRATION_TESTING  # noqa: E501
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)